src/Resources/public/js/fragmentList.jquery.js

```diff
@@ -184,31 +188,91 @@
 
             // Remove a persisted fragment (just check delete checkbox)
             if (!$frag.hasClass('is-tmp')) {
-                // Check the delete checkbox
-                $cb.prop('checked', true).attr('checked', true);
-
                 // Hide form block
                 $form.hide();
-                // Remove a new fragment (not already saved)
+                this.moveSelectedFragmentOutsideOfForm(id, $frag, $form);
+
+            // Remove a new fragment (not already saved)
             } else {
+                $frag.remove();
                 $form.remove();
                 this.setFormListElement();
             }
 
-            // Remove list element
-            $frag.remove();
-
-            // Removed the last element ?
+            //Removed the last element ?
             if(!this.$list.children().length) {
-                this.$activeFragment = null;
-                // Removed the current visible item ?
+               this.$activeFragment = null;
+
+            // Removed the current visible item ?
             } else if(id === this.$activeFragment.data('fragId')) {
-                this.setActive(this.$list.children(':first-child'));
+               this.setActive(this.$list.children(':first-child'));
             }
 
             // Reorder
             this.resetOrder();
+        },
+
+        /**
+         * This method move the selected fragment and from elements in div#deletedFragments
+         *
+         * @param id
+         * @param fragmentElement
+         * @param formElement
+         */
+        moveSelectedFragmentOutsideOfForm: function (id, fragmentElement, formElement) {
+            var divElement = $('<div>')[0];
+            divElement.setAttribute('id', id);
+            fragmentElement.clone().appendTo(divElement);
+            divElement.append(formElement[0]);
+
+            // Generate the elements who contains the fragment title and the cancel button.
+            var resetRemovedElement = $('<span>');
+            var fragmentTitle = fragmentElement.find('.fragment__title').text();
+
+            var cancelIcon = $('<i>');
+            cancelIcon.attr('class', 'fa fa-repeat');
+            cancelIcon.on('click', this.cancelFragmentDeletion);
+
+            resetRemovedElement.append(cancelIcon);
+            resetRemovedElement.append(' <i>' +  fragmentTitle + '</i>');
+
+            // Insert the title of fragment and the reset button at the pace of removed fragment.
```

Comment:
`Insert the fragment's title and the reset button in the place of removed fragment.`?